TWLight/applications/views.py

```diff
@@ -408,17 +409,18 @@ def get_success_url(self):
         messages.add_message(
             self.request,
             messages.SUCCESS,
+            # fmt: off
             # Translators: When a user applies for a set of resources, they receive this message if their application was filed successfully.
-            _(
-                "Your application has been submitted for review. "
-                'Head over to <a href="{applications_url}">My Applications'
-                "</a> to view the status.".format(
-                    applications_url=reverse_lazy(
-                        "users:my_applications",
-                        kwargs={"pk": self.request.user.editor.pk},
-                    )
+            _("Your application has been submitted for review. "
+              'Head over to <a href="{applications_url}">My Applications'
+              "</a> to view the status.".format(
+                applications_url=reverse_lazy(
+                    "users:my_applications",
+                    kwargs={"pk": self.request.user.editor.pk},
                 )
+              )
             ),
```

Comment:
closing brackets for `_()` should be lifted to the same line as the source string